tests/test-cases/Group1-Docker-Commands/1-23-Docker-Inspect.robot

```diff
@@ -155,3 +155,12 @@ Docker inspect container status
     # keyword at top of file
     ${stopped}=  Get container inspect status  ${container}
     Should Contain  ${stopped}  exited
+
+Docker inspect container with specified DNS
+    ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --dns=8.8.8.8 --name=test-with-specified-dns busybox sleep 600
```

Comment:
Can we extend this test case to show support for multiple DNS servers? The implementation appears to handle that case, so it would be good to test as well to prevent regressions in the future.

Reply:
Cool, good advice, let me add it!
Yes, the implementation support multiple dns servers. There are two cases:
1> The container is attached to one network, here we can specify mutiple DNS servers
2> The container is attached to two networks, for example bridge and container network, here the DNS servers for these two networks are all configured to resolv.conf